satellite/console/auth.go

```diff
@@ -66,7 +66,7 @@ func GetAuth(ctx context.Context) (Authorization, error) {
 	}
 
 	if _, ok := value.(error); ok {
-		return Authorization{}, errs.New(internalErrMsg)
+		return Authorization{}, ErrConsoleInternal.New("auth key can not be parsed")
```

Comment:
This error provide information about why auth failed it should not be modified

Reply:
Previous error didn't differs from general ``Internal error``. This implementation specify what happened